ci/L0_backend_vllm/test.sh

```diff
@@ -45,6 +47,16 @@ mkdir -p models/add_sub/1/
 cp ../qa_models/add_sub/model.py models/add_sub/1/
 cp ../qa_models/add_sub/config.pbtxt models/add_sub
 
+# Invalid model attribute
+mkdir -p models/vllm_invalid_1/1/
+cp ../qa_models/vllm_opt/config.pbtxt models/vllm_invalid_1/
```

Comment:
Same here, could you please use `samples` directory and copy file from there, then you can just change one of entries to json to an invalid attribute with `sed`: https://www.geeksforgeeks.org/sed-command-in-linux-unix-with-examples/

Reply:
Updated tests.